Dockerfile

```
@@ -491,45 +453,115 @@ ENV CGO_CFLAGS="${CFLAGS}"
 ENV CGO_CXXFLAGS="${CXXFLAGS}"
 ENV CGO_LDFLAGS="${LDFLAGS}"
 
+ENV GO111MODULE="auto"
+
+ENV AWS_LC_FIPS_VER="2.0.9"
+
+USER root
+RUN dnf -y install golang
+
+ENV GO121VER="1.21.9"
+ENV GO122VER="1.22.1"
+
+# =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+FROM sdk-go-prep as sdk-go-1.21-prep
+
+USER builder
+
+WORKDIR /home/builder/sdk-go-${GO121VER}
+
+COPY ./hashes/go-1.21 /home/builder/hashes-go
+COPY ./helpers/go/* ./
+COPY ./patches/go-1.21 /home/builder/patches-go
+
+COPY ./hashes/aws-lc /home/builder/hashes-aws-lc
+COPY ./patches/aws-lc /home/builder/patches-aws-lc
+
+RUN ./prep-go.sh --go-version=${GO121VER} 
```

With the suggestion above, this would become:
```suggestion
COPY ./hashes/go-${GOREL} /home/builder/hashes-go
COPY ./helpers/go/* ./
COPY ./patches/go-${GOREL} /home/builder/patches-go

COPY ./hashes/aws-lc /home/builder/hashes-aws-lc
COPY ./patches/aws-lc /home/builder/patches-aws-lc

RUN ./prep-go.sh --go-version=${GOVER} 
```